stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -77,6 +77,21 @@
 
     static final String API_VERSION = "2017-06-05";
 
+    // Used for HTTP-level tests
+    private static @Nullable MockGetStripeResponse mockGetStripeResponse;
+
+    interface MockGetStripeResponse{
+        StripeResponse getStripeResponse(@RestMethod String method,
```

Comment:
Can you annotate all these argument params and return accordingly? (e.g. `@NonNull`)